cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -413,22 +412,7 @@ func (command *Evaluate) evaluateLocal(evaluationContext *evaluate.Context) (err
 	}
 
 	assessments, totalScore := evaluate.Evaluate(evaluationContext)
-
 	assessmentsPerModel := assessments.CollapseByModel()
-	if err := (report.Markdown{
```

Comment:
Please move this commit into a separate PR. Not 100% sure if we want to scrap that yet. Good to have. But will not merge that yet.